Formula/packetbeat.rb

```diff
@@ -15,26 +15,46 @@ class Packetbeat < Formula
 
   depends_on "go" => :build
 
+  resource "virtualenv" do
+    url "https://files.pythonhosted.org/packages/d4/0c/9840c08189e030873387a73b90ada981885010dd9aea134d6de30cd24cb8/virtualenv-15.1.0.tar.gz"
+    sha256 "02f8102c2436bb03b3ee6dede1919d1dac8a427541652e5ec95171ec8adbc93a"
+  end
+
   def install
-    gopath = buildpath/"gopath"
-    (gopath/"src/github.com/elastic/beats").install Dir["{*,.git,.gitignore}"]
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/elastic/beats").install buildpath.children
+
+    ENV.prepend_create_path "PYTHONPATH", buildpath/"vendor/lib/python2.7/site-packages"
+
+    resource("virtualenv").stage do
+      system "python", *Language::Python.setup_install_args(buildpath/"vendor")
+    end
 
-    ENV["GOPATH"] = gopath
+    ENV.prepend_path "PATH", buildpath/"vendor/bin"
 
-    cd gopath/"src/github.com/elastic/beats/packetbeat" do
+    cd "src/github.com/elastic/beats/packetbeat" do
+      # prevent downloading binary wheels
+      inreplace "../libbeat/scripts/Makefile", "pip install", "pip install --no-binary :all"
```

Comment:
There needs to be an upstream fix for this in the Makefile with a variable like `PIP_INSTALL_PARAMS` or similar.

Reply:
I opened elastic/beats#5693. Can we pull this in now (as this will not be available upstream until 6.0.1)?

Reply:
Thanks for taking care of that :)